localstack/services/apigateway/provider.py

```diff
@@ -667,6 +668,21 @@ def import_rest_api(
             ),
         )
 
+    def delete_integration(
+        self, context: RequestContext, rest_api_id: String, resource_id: String, http_method: String
+    ) -> None:
+
+        try:
+            return _call_moto(
+                context,
+                "DeleteIntegration",
```

Comment:
Nit: I think here we could simply use the `call_moto(..)` callback, no need to pass the operation name.

Also, wondering if the try/except is required - ASF should already be able to serialize error responses raised from within call_moto - or is this a special case of an incompatible error coming from moto?

Reply:
Makes sense, I'll double check! 👍

Reply:
Checked, without the try-catch block, 
```
An error occurred (500) when calling the DeleteIntegration operation (reached max retries: 2):
```

with the try-catch
```
An error occurred (NotFoundException) when calling the DeleteIntegration operation: Invalid Resource identifier specified
```
which is the correct one.